wrappers/rust/icicle-core/src/ntt/tests.rs

```diff
@@ -50,34 +50,38 @@ where
 {
     let test_sizes = [1 << 4, 1 << 17];
     for test_size in test_sizes {
-        let ark_domain = GeneralEvaluationDomain::<F::ArkEquivalent>::new(test_size).unwrap();
+        for alg in [NttAlgorithm::Radix2, NttAlgorithm::MixedRadix] {
+            let ark_domain: GeneralEvaluationDomain<<F as ArkConvertible>::ArkEquivalent> =
```

Comment:
Probably rust-analyzer did it for me by mistake. Generally I think type annotations are a good thing but in this case I absolutely agree that it is redundant to annotate since the constructor is clearly describing the type.
I will remove it. Did not intend to add that.